hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceConfigurationBuilder.java

```diff
@@ -496,7 +528,7 @@ private static void determineMetricSetPrometheus(ModelNode config,
 
                     MetricType<PlatformNodeLocation> usableSpace = new MetricType<PlatformNodeLocation>(null,
                             Constants.FILE_STORE_USABLE_SPACE,
-                            new AttributeLocation<PlatformNodeLocation>(
+                            new AttributeLocation<>(
```

Comment:
This is one of the syntax sugar changes I made while I was browsing the code. Most of those changes are not related and I intend to remove them, unless the Agent is meant to work on Java8+ only, in which case those changes are harmless.